File: Python/백준/Bronze/30454. 얼룩말을 찾아라！/얼룩말을 찾아라！.py
a,b = map(int,input().split())
aa = []
for i in range(a):
    s = input()
    cnt = 0
    for j in range(b):
        if s[j]=='1':
            if j==0 or s[j-1]=='0':
                cnt+=1
    aa.append(cnt)
print(max(aa),aa.count(max(aa)))

File: C#/백준/Bronze/30454. 얼룩말을 찾아라！/얼룩말을 찾아라！.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int a = aa[0];
            int b = aa[1];
            int l = 0;
            int n = 0;
            for (int i=0;i<a;i++){
                int cnt = 0;
                String s = Console.ReadLine();
                for (int j=0;j<b;j++){
                    if (s[j]=='1' && (j==0 || s[j-1]=='0')){
                        cnt++;
                    }
                }
                if (l<cnt){
                    n = 1;
                }
                else if(l==cnt){
                    n++;
                }
                l = l>cnt?l:cnt;
            }
            Console.WriteLine($"{l} {n}");
        }
    }
}